pylint/checkers/utils.py

```diff
@@ -1690,3 +1693,13 @@ def returns_bool(node: nodes.NodeNG) -> bool:
         and isinstance(node.value, nodes.Const)
         and node.value.value in {True, False}
     )
+
+
+def get_node_first_ancestor_of_type(
+    node: nodes.NodeNG, ancestor_type: Tuple[Type[T_Node]]
+) -> Optional[T_Node]:
```

Comment:
Don't know if that has been considered. There is no need to limit `ancestor_type` to `Tuple[...]`. `isinstance` works fine with `Type[T_Node]` too. Ie.
```py
ancestor_type: Union[Type[T_Node], Tuple[Type[T_Node]]]
```
This has the advantage that now you can pass both a single type and a tuple of types to it.